pkg/types/azure/defaults/machines.go

```diff
@@ -14,10 +14,11 @@ func BootstrapInstanceType(region string) string {
 
 // ControlPlaneInstanceType sets the defaults for control plane instances.
 // Minimum requirements are 4 CPU's, 16GiB of ram, and 120GiB storage.
```

Comment:
If we're raising the type here to keep etcd happy, why aren't we bumping these minimum requirements too?  Isn't "happy etcd under e2e" the lowest bar?